### PR TITLE
docs:created reporting guidelines

### LINKED ISSUE #1048: Create a reporting_guidelines.md
## Description
As a user, I need to modify the reporting guidelines, so that I can seek for help when necessary.
1. The content should be consistent with [Reporting Guidelines](https://github.com/anitab-org/mentorship-flutter/blob/develop/.github/reporting_guidelines.md)
2. And then update the code of conduct link in **Readme.md**>>Contributing section.
3. Add the updated Reporting Guidelines file to doc folder.

## Acceptance Criteria
- [ ] Update [Required]

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
30 minutes

## Note
This issue will be assigned from September 1, 2020

### PR DESCRIPTION
**Pull request by vj-codes**
# Description
Created and updated reporting_guidelines.md

Fixes #1048 

# Type of Change:
**Delete irrelevant options.**

- Documentation

# Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by sanchibansal340 — COMMENTED** [review]
Looks good to me

**Review by sanchibansal340 — DISMISSED** [review]

**Review by keshakaneria — CHANGES_REQUESTED** [review]
Looks good to me! Just squash the commits before ready to merge, otherwise its well 👍

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ production at the moment. It may be configured to do so in the future.
 Contribute
 ----------
 
-- Please read our [Contributing guidelines](CONTRIBUTING.md), [Code of Conduct](code_of_conduct.md) and [Reporting Guidelines](reporting_guidelines.md)
+- Please read our [Contributing guidelines](CONTRIBUTING.md), [Code of Conduct](code_of_conduct.md) and [Reporting Guidelines](docs/reporting_guidelines.md)
```
Comment (SanketDG) [inline README.md:76]:
```suggestion
- Please read our [Contributing guidelines](CONTRIBUTING.md), [Code of Conduct](CODE_OF_CONDUCT.md) and [Reporting Guidelines](docs/reporting_guidelines.md)
```

**Comment by sanchibansal340** [comment]
@vj-codes Can you please check the suggested changes? :)

File: README.md (hunk 1 above)
Comment (vj-codes) [reply README.md:76]: @SanketDG @sanchi0204 hey I don't see a CODE_OF_CONDUCT file in docs folder and the one which is there is already in the same format unlike the suggested change which after changing will lead to an error.Also this issue was about reporting guidelines so please let me know if I should change it

Comment (SanketDG) [reply README.md:76]: Feel free to rename it though, it's a simple `git mv` command. :slightly_smiling_face:

**Comment by sanchibansal340** [comment]
@vj-codes Any updates?

**Comment by vj-codes** [comment]
@sanchibansal340 hello sorry for the late reply I almost forgot about this PR.
I have updated as per changes requested and since this was a documentation issue I used online editor so can't really squash the commits

**Review by keshakaneria — APPROVED** [review]
Looks good now as per the changes requested 👍

**Review by SanketDG — APPROVED** [review]

**Pull request merged**